packages/typespec-azure-resource-manager/src/commontypes.private.decorators.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  DecoratorContext,
+  Enum,
+  EnumValue,
+  Model,
+  ModelProperty,
+  Program,
+} from "@typespec/compiler";
+import { ArmStateKeys } from "./state.js";
+
+export const namespace = "Azure.ResourceManager.CommonTypes.Private";
```

Comment:
I don't think we can remove the existing decorators from the existing namespace, this has been used by folks to create common-types that we did not yet have coverage for

Reply:
I am little concern about same name under different namespaces. Did a search, none was found in public main and RPSaaSMaster